crates/primitives/src/lib.rs

```diff
@@ -1,5 +1,4 @@
-#![deny(unused_crate_dependencies)]
```

Comment:
When "abi" is not set, it assumes that `schemars` is unused, even though it's set as an optional feature

Reply:
try `cargo near build reproducible-wasm` for contract

Reply:
@kuksag I think this still holds. Can you add it before merging?